website/static/js/pages/home-page.js

```diff
@@ -1,216 +1,110 @@
-var TweenLite = require('TweenLite');
-require('EasePack');
-require('vendor/youtube');
-
-// ANIMATION FOR FRONT PAGE
-$( document ).ready(function() {
-    $('#logo').removeClass('off');
-    $('.youtube').YouTubeModal({autoplay:1, width:640, height:480});
-});
-
-var waitForFinalEvent = (function () {
-    var timers = {};
-    return function (callback, ms, uniqueId) {
-        if (!uniqueId) {
-          uniqueId = 'Don\'t call this twice without a uniqueId';
-        }
-        if (timers[uniqueId]) {
-          clearTimeout (timers[uniqueId]);
-        }
-        timers[uniqueId] = setTimeout(callback, ms);
-    };
-})();
-
-(function() {
-
-    var width;
-    var height;
-    var largeHeader;
-    var canvas;
-    var ctx;
-    var points;
-    var target;
-    var animateHeader = true;
-
-    // Main
-    initHeader();
-    initAnimation();
-    addListeners();
-
-    function initHeader() {
-        width = window.innerWidth;
-        height = 800;
-        target = {
-            x: width/2,
-            y: 300
-        };
-
-        largeHeader = document.getElementById('home-hero');
-
-        canvas = $('#demo-canvas')[0];
-        canvas.width = width;
-        canvas.height = height;
-        ctx = canvas.getContext('2d');
-
-        // create points
-        points = [];
-        for(var x = 0; x < width; x = x + width/20) {
-            for(var y = 0; y < height; y = y + height/20) {
-                var px = x + Math.random()*width/20;
-                var py = y + Math.random()*height/20;
-                var p = {x: px, originX: px, y: py, originY: py };
-                points.push(p);
-            }
-        }
-
-        // for each point find the 5 closest points
-        for (var i = 0; i < points.length; i++) {
-            var closest = [];
-            var p1 = points[i];
-            for (var j = 0; j < points.length; j++) {
-                var p2 = points[j];
-                if (p1 !== p2) {
-                    var placed = false;
-                    for (var k = 0; k < 5; k++) {
-                        if (!placed) {
-                            if(closest[k] === undefined) {
-                                closest[k] = p2;
-                                placed = true;
-                            }
-                        }
-                    }
-                    for (var m = 0; m < 5; m++) {
-                        if (!placed) {
-                            if (getDistance(p1, p2) < getDistance(p1, closest[m])) {
-                                closest[m] = p2;
-                                placed = true;
-                            }
-                        }
+/**
+ * Initialization code for the home page.
+ */
+
+'use strict';
+
+var $ = require('jquery');
+var m = require('mithril');
+
+var quickSearchProject = require('js/quickProjectSearchPlugin');
+var newAndNoteworthy = require('js/newAndNoteworthyPlugin');
+var meetingsAndConferences = require('js/meetingsAndConferencesPlugin');
```

Comment:
Minor: to make things more cohesive, it might make sense to put all these home page components in their own folder, e.g. `js/home-page/quickSearch`, `js/home-page/newAndNoteworthy`.